src/views/Releases/ListReleases/index.jsx

```diff
@@ -215,8 +267,26 @@ function ListReleases(props) {
     });
   };
 
-  const handleReadOnlySubmit = async () => {
-    const release = dialogState.item;
+  const scheduleReadWriteChange = async release => {
+    const sc = {
+      change_type: 'update',
+      when: new Date().getTime() + 5000,
+      name: release.name,
+      product: release.product,
+      read_only: false,
+      data_version: release.data_version,
+    };
+    const { data, error } = await addScheduledChange(sc);
+    const { data: scData } = await getScheduledChangeById(data.sc_id);
```

Comment:
I think this will throw a 404 if `addScheduledChange` threw an error (`sc_id` won't exist in the backend). I think you should also check for an error from `getScheduledChangeById` throw if it exists, otherwise errors from it will be hidden.